api/test/e2e/global_rule_test.go

```diff
@@ -282,3 +288,108 @@ func TestGlobalRule(t *testing.T) {
 		testCaseCheck(tc, t)
 	}
 }
+
+func TestGlobalRule_with_createtime_updatetime(t *testing.T) {
+	tests := []HttpTestCase{
+		{
+			Desc:   "create global rule",
+			Object: ManagerApiExpect(t),
+			Path:   "/apisix/admin/global_rules/1",
+			Method: http.MethodPut,
+			Body: `{
+				     "plugins": {
+					   "response-rewrite": {
+					     "headers": {
+						   "X-VERSION":"1.0"
+						 }
+					   },
+					   "uri-blocker": {
+					     "block_rules": ["select.+(from|limit)", "(?:(union(.*?)select))"]
+					   }
+					 }
+				   }`,
```

Comment:
@starsz ok, I will revert the styles here, as well as keep the same afterwards. Thanks.

Reply:
@starsz current codes mixed spaces and tabs here. So honestly, the existing style only seems good in GitHub pages, but not in VSCode editor. Please see the screenshot below:

<img width="664" alt="image" src="https://user-images.githubusercontent.com/2542401/103971535-d6caca00-51a5-11eb-9130-7d8491549a89.png">

Shall we use `jq` to format the json and paste it into code? Looking forward to your feedback. Thanks.

Reply:
Paste it into code is OK. Just make sure that we should use a table rather than space.
Thanks a lot. 

See other Code:
https://github.com/apache/apisix-dashboard/blob/40d8f97de8770187599ff5963617049603020e92/api/test/e2e/label_test.go#L56-L69

Reply:
Sorry, I change my mind. Both are OK, just keep the brackets align.

Reply:
@starsz much appreciated to your kindness. PR renewed and I followed the style of existing code. I would prefer not to discuss the style stuff here. 😄